src/contributors/surbhirathi.json

```
@@ -0,0 +1,7 @@
+  
+{
+    "name": "Surbhi Rathi", 
+    "desc": "Web Developer",
+    "github": "https://github.com/SurbhiRathi23/",
```

```suggestion
    "github": "https://github.com/SurbhiRathi23/"
```
Remove this trailing comma.